src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -27,6 +27,7 @@
     public CommandBox(CommandExecutor commandExecutor) {
         super(FXML);
         this.commandExecutor = commandExecutor;
+
         // calls #setStyleToDefault() whenever there is a change to the text of the command box.
```

Comment:
Perhaps this should start with capital letter?
```suggestion
        // Calls #setStyleToDefault() whenever there is a change to the text of the command box.
```